src/View.js

```diff
@@ -0,0 +1,47 @@
+//@ts-check
+const { GAME_STATE } = require("./enum");
+
+class View {
+	static #PROMPT_ING = "숫자를 입력해주세요 : ";
+  static #PROMPT_END = "게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.\n";
+	static #MESSAGE_START = "숫자 야구 게임을 시작합니다.";
+	static #MESSAGE_END = "3개의 숫자를 모두 맞히셨습니다! 게임 종료";
+	constructor(ingHandler, endHandler) {
+    this._console = require("@woowacourse/mission-utils").Console;
+    this._ingHandler = ingHandler;
+		this._endHandler = endHandler;
+  }
+
+	/** @param {GAME_STATE} state */
+	input(state) {
+		if (state === GAME_STATE.END) {
+			this._console.readLine(View.#PROMPT_END, (command) => {
+				this._endHandler(command);
```

Comment:
혹시 익명함수로 감싸는 것 말고 아래와같이 바인딩도 고려해보셨었나요!?:)
```suggestion
			this._console.readLine(View.#PROMPT_END, this._endHandler.bind(this))
```

Reply:
ㅎㅎ 넵!! bind가 싫으시면 _endHandler를 화살표함수로 만들어서 프로퍼티 함수로 만드는 것도 방법일 거에요!
프로터피 함수 vs method의 차이는 알아야할 것 같은데 저도 정확하게 알지 못해서!.. 좀더 공부해봐야겠어요:)